apps/src/templates/SpeedSlider.jsx

```diff
@@ -0,0 +1,233 @@
+var React = require('react');
+var color = require('../color');
+var Radium = require('radium');
+var dom = require('../dom');
```

Comment:
Nit: I'd be nice to use ES6-style imports for these, `import React from 'react';` etc.  Might need to be careful importing dom.js that way though, it's using nonstandard export syntax.